inference-engine/samples/hello_classification/main.cpp

```diff
@@ -130,6 +130,7 @@ int main(int argc, char *argv[]) {
 
         // --------------------------- 8. Process output ------------------------------------------------------
         Blob::Ptr output = infer_request.GetBlob(output_name);
+        if (network.getOutputsInfo().size() != 1) throw std::logic_error("Sample supports topologies with 1 output only");
```

Comment:
It is better to perform such a check right after the reading of the model:  
CNNNetwork network = ie.ReadNetwork(input_model);